packages/tools/fluid-runner/src/index.ts

```diff
@@ -4,15 +4,21 @@
  */
 
 /* eslint-disable import/no-internal-modules */
-export { ICodeLoaderBundle, IFluidFileConverter } from "./codeLoaderBundle";
-export { createContainerAndExecute, exportFile, IExportFileResponse } from "./exportFile";
-export { fluidRunner } from "./fluidRunner";
-export { OutputFormat, ITelemetryOptions } from "./logger/fileLogger";
+export { ICodeLoaderBundle, IFluidFileConverter } from "./codeLoaderBundle.js";
+export {
+	createContainerAndExecute,
+	exportFile,
+	IExportFileResponse,
+	IExportFileResponseSuccess,
+	IExportFileResponseFailure,
+} from "./exportFile.js";
+export { fluidRunner } from "./fluidRunner.js";
+export { OutputFormat, ITelemetryOptions, IFileLogger } from "./logger/fileLogger.js";
```

Comment:
I see exports were added to resolve api-extractor issues. Do we know the history for the prior choices? ...to go against api-extractor and not export those? Such changes are best done explicitly and separately.

Reply:
api-extractor forgotten-export warning were suppressed before: https://github.com/microsoft/FluidFramework/pull/20349/files#diff-04ef5f87a3b831c02ccb64223b9bfc0bb33d0403d8b2f250179dbaf9e50267f3